agents/kubevirt/fence_kubevirt.py

```diff
@@ -17,9 +17,10 @@ def get_nodes_list(conn, options):
     logging.debug("Starting list/monitor operation")
     result = {}
     try:
+        apiversion = options.get("--apiversion")
         namespace = options.get("--namespace")
         include_uninitialized = True
-        vm_api = conn.resources.get(api_version=API_VERSION, kind='VirtualMachine')
+        vm_api = conn.resources.get(api_version=apiversion, kind='VirtualMachine')
```

Comment:
You should also remove the hardcoded API_VERSION on line 14.

Reply:
sure, just seen it as well. thanks. I pushed an updated commit